source/includes/stackpath/_images.md

```diff
@@ -0,0 +1,106 @@
+## Images
+
+StackPath Edge Computing makes use of images to deploy virtual machines within a workload. The image can be from StackPath default images or
+a custom one. Only non-deprecated images are returned by default.
+
+Deploy and manage your Images.
```

Comment:
should be "your images" in lowercase

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/268/commits/bc7606f72cc3b43fea9d39f4c1b787a147714c17